filters/annoyances.txt

```diff
@@ -6831,6 +6831,10 @@ ph.ontools.net##+js(acs, document.oncontextmenu)
 ! https://github.com/uBlockOrigin/uAssets/issues/18048
 neilpatel.com##+js(no-xhr-if, /hotjar|googletagmanager/)
 
+! Youtube warning
+youtube.com##+js(json-prune, auxiliaryUi.messageRenderers.enforcementMessageViewModel)
+youtube.com##+js(set, ytInitialPlayerResponse.auxiliaryUi.messageRenderers.enforcementMessageViewModel, undefined)
```

Comment:
Why not `,youtubekids.com,youtube-nocookie.com`?

Reply:
no one has reported these sites yet